packages/swirl-components/src/components/swirl-autocomplete/swirl-autocomplete.tsx

```diff
@@ -37,32 +37,34 @@ export type SwirlAutocompleteSuggestion = {
   styleUrl: "swirl-autocomplete.css",
   tag: "swirl-autocomplete",
 })
-export class SwirlAutocomplete implements SwirlFormInput {
+export class SwirlAutocomplete implements SwirlFormInput<string | string[]> {
   @Element() el: HTMLElement;
 
   @Prop() autoSelect?: boolean;
   @Prop() clearable?: boolean = true;
   @Prop() clearButtonLabel?: string = "Clear input";
   @Prop() disabled?: boolean;
```

Comment:
It seems this prop does not work (anymore). When i hit disabled in storybook i still can interact with the component

Reply:
Fixed.